llvm/test/MC/AArch64/arm64-elf-relocs.s

```diff
@@ -35,63 +35,75 @@
 // CHECK: add x5, x0, :tlsdesc_lo12:sym
 // CHECK-OBJ-LP64: 1c R_AARCH64_TLSDESC_ADD_LO12 sym
 
+   add x5, x0, #:tlsdesc_auth_lo12:sym
```

Comment:
You can move this to a new section so that the section offsets below won't be adjusted

Reply:
@MaskRay I moved new test cases to the end of the file so previous offsets do not get changed.

Please let me know if this resolves your comment.